lib/validator.js

```diff
@@ -49,6 +49,9 @@ const validators = {
   is(str, pattern, modifiers) {
     return this.regex(str, pattern, modifiers);
   },
+  notEmpty(str) {
+    return !str.match(/^[\s\t\r\n]*$/);
```

Comment:
no need to match the complete string, test for non whitespace character shall suffice

```suggestion
    return /[^\s]/.test(str);
```

Reply:
btw, `\s` includes `[\t\r\n]`